### ISSUE TITLE
Clearly map EPUB a11y features to importance / conformance levels 

### ISSUE THREAD

**Issue by llemeurfr** [w3c/epub-specs#1289]
When reading "EPUB accessibility" and "EPUB accessibility techniques", the EPUB specific functionalities (presence of landmarks, media overlays ...) are not prioritized nor mapped to a notion of conformance level. 

Can we expand the A/AA/AAA WCAG notion to EPUB features?
Should we instead another way to express the importance of enhancing an EPUB file with a feature or another?

**Comment by dauwhe** [comment]
I would be very wary of using WCAG terminology for something that is not officially part of WCAG.

**Comment by mattgarrish** [comment]
Right, that's why we assiduously avoided making any comparisons like that, even to the point of separating WCAG conformance from overall spec conformance. The accessibility specification spells out the normative requirements for including the epub features.

That said, we are still working to get a couple of success criteria added around page breaks. For the latest effort, see https://lists.w3.org/Archives/Public/w3c-wai-gl/2019JulSep/0033.html

**Comment by llemeurfr** [comment]
I understand there may be a logical issue inside W3C groups if the same A to AAA codes were to be reused; but it would be much simpler to understand by publishers, who do not consider WCAG by itself, but WCAG inside EPUB 3. 

@dauwhe, @mattgarrish, if this A to AAA categorization system is not chosen by the community, do you mean you don't want to consider ANY way of giving relative importance to the different EPUB accessibility features described by the accessibility spec?

**Comment by avneeshsingh** [comment]
The main accessibility features are:
Accessibility metadata
Page markers
Media Overlays

Media Overlays is non-normative in version 1.0, we will be working on it further in next version. So, there is no need of assigning levels to it at this point of time.

Accessibility metadata is essential requirement.
Page break markers is also essential requirement when there are pages in print equivalent or fixed layout equivalent.

So, level A, AA, AAA differentiation may not be practical in current situation.

**Comment by llemeurfr** [comment]
@avneeshsingh from your comment I infer that "everything is equally important". 

I don't want to interfere with discussions to be held elsewhere, so I'll just comment to illustrate why giving an importance to different a11y features is a MUST: Media Overlays are not something trade publishers will add to their ebooks, even if they want their ebooks to be highly accessible. And I heard from several people involved in a11y efforts that Media Overlays were not so great priority for them (in comparison with other features), as audiobooks and proper TTS can "make the trick" for some time. 

So when the time comes to make media overlays normative, a proper conformance level will have to be applied IMO.

**Comment by avneeshsingh** [comment]
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/w3c/publ-epub-revision/issues/1289?email_source=notifications\u0026email_token=ADJDWTXJ6DCBWHIOMU6CAMTP7R2FXA5CNFSM4IDU4AX2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODZ5U66I#issuecomment-511397753",
"url": "https://github.com/w3c/publ-epub-revision/issues/1289?email_source=notifications\u0026email_token=ADJDWTXJ6DCBWHIOMU6CAMTP7R2FXA5CNFSM4IDU4AX2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODZ5U66I#issuecomment-511397753",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by mattgarrish** [comment]
> I'll just comment to illustrate why giving an importance to different a11y features is a MUST

The "Meeting this objective" sections already define what level of conformance these additional requirements have.

The requirements are not part of WCAG, and in discussions with W3C when we created the specification we agreed to avoid exactly what you're suggesting because it would give the false impression that the requirements are part of meeting WCAG, or that somehow we're taking over and redefining WCAG for publishing.

**Comment by avneeshsingh** [comment]
Regarding concerns about Media Overlays, we understand that all the publishers cannot be expected to provide Media Overlays, this is why we decided not to do it in rush in EPUB Accessibility version 1.0. We will not ignore this fact when we will be working on the next version.

**Comment by laudrain** [comment]
If I understand well, Media Overlays are meaningful only if the EPUB contains a sound file of the text content. If there is no such audio file, there is no Media Overlays.
The European Accessibility Act is on the same base : Media Overlay is mandatory IF there is an audio file of the text.

**Comment by llemeurfr** [comment]
Reading again the documents, I see that the section about Media Overlays in EPUB Accessibility is not normative and not mapped to a sibling section in the EPUB Accessibility Techniques: as @avneeshsingh says, there is no rush for that aspect. 

Ok so the model expressed in EPUB Accessibility (2.2) is that an EPUB is "accessible" if, among other requirements, if it "meets the requirements for EPUB Publications" -> page navigation only. 

In this case I understand why there is no need to create levels of conformance for now. 

I suppose we can state things like "my EPUB 3 publications are accessible and their resources are conformant with WCAG AA".

**Comment by murata2makoto** [comment]
It would be very nice if we can make something comparable to [the poster from GOV.UK](https://github.com/UKHomeOffice/posters/blob/master/accessibility/dos-donts/posters_en-UK/accessibility-posters-set.pdf) for the accessibility of EPUB publications.

**Comment by avneeshsingh** [comment]
I remember that Matt wrote a document of similar kind on IDPF side. And later on BISG and ABC came up with quick start guides.
https://inclusivepublishing.org/toolbox/accessibility-guidelines/


I remember our discussion about Japanese specific techniques (Ruby characters etc.), in DPUB summit. I think we should figure it out while revising the EPUB accessibility techniques. A couple of weeks ago, I opened an issue for getting inputs for updating the techniques. It would be great if you can describe the issue related to Ruby on the following issue, or may be open a new issue for it.
https://github.com/w3c/publ-cg/issues/89

**Comment by murata2makoto** [comment]
> A couple of weeks ago, I opened an issue for getting inputs for updating the techniques. It would be great if you can describe the issue related to Ruby on the following issue, or may be open a new issue for it.
w3c/publ-cg#89

I started to solicit comments.  In particular, I gave a talk in a Japanese Daisy Consortium seminar last week, mentioned the upcoming revision of the EPUB accessibility techniques, and solicited comments.  But it will take time (probably by the end of this year for ruby accessibility).  Probably a nice topic for TPAC.

**Comment by mattgarrish** [comment]
Closing this issue as assigning WCAG levels is problematic for the reasons already discussed.

As I've raised in #1503, though, perhaps there's a way of tiering the requirements based on the wcag level that is being met. We'll see how that works out.
